client/gutenberg/extensions/tiled-gallery/constants.js

```diff
@@ -30,4 +30,5 @@ export const LAYOUT_STYLES = [
 	*/
 ];
 export const MAX_COLUMNS = 20;
+export const MOSAIC_MARGIN = 4;
```

Comment:
Wouldn't this be the margin for all the layouts, including columns?

Reply:
I suppose so

Reply:
I think it should be `GUTTER` as well.